packages/edit-site/src/index.js

```diff
@@ -84,6 +85,11 @@ export function initializeList( id, templateType, settings ) {
 		defaultTemplatePartAreas: settings.defaultTemplatePartAreas,
 	} );
 
+	// Default the navigation panel to be opened when we're in a bigger screen.
+	dispatch( editSiteStore ).setIsNavigationPanelOpened(
+		select( viewportStore ).isViewportMatch( 'medium' )
```

Comment:
There's a `useViewportMatch` hook in the compose package, I don't think we need to bring in the viewport dependency directly.

Reply:
I want to synchronously update the store's default value before rendering the React tree. So that we don't have to put it inside an `useEffect` to schedule it after the first render, causing unnecessary re-render and jump of UI.

Reply:
Ok, I see. Might be worth adding a comment here to explain that so that someone like me doesn't refactor it 😄

Reply:
Sure, good idea.